test/js/lib/validate-remote-file-content.test.js

```diff
@@ -0,0 +1,107 @@
+/**
+ * @fileoverview Tests for client-side file validation (CSFV) for sync
+ */
+
+import util from 'util'
+
+import fetch, { Headers } from 'node-fetch'
+import '@testing-library/jest-dom/extend-expect'
+
+import {
+  validateRemoteFileContent,
+  MAX_SYNC_CSFV_BYTES, updateSyncProps, getFileFromRangeData
+} from 'lib/validation/validate-remote-file-content'
+
+describe('Client-side file validation for sync', () => {
+  beforeAll(() => {
+    global.fetch = fetch
+    global.Headers = Headers
+  })
+
+  it('sends range request for remote file', async () => {
+    const fakeLog = jest.spyOn(global, 'fetch')
+    fakeLog.mockImplementation(() => { })
+
+    window.SCP = {
+      readOnlyToken: 'test'
+    }
+
+    const bucketName = 'broad-singlecellportal-public'
```

Comment:
pull this into a constant at the top of the file -- we'll want it for other tests soon probably

Reply:
Good point, done!